tests/nested/manual/uc20-install-in-initrd/task.yaml

```diff
@@ -3,9 +3,22 @@ summary: Check installation in initrd
 systems: [ubuntu-20.04-64]
 
 environment:
-  NESTED_UBUNTU_IMAGE_PRESEED_KEY: "\" (test)\""
-  NESTED_ENABLE_TPM: false
-  NESTED_ENABLE_SECURE_BOOT: false
+  INSTALL_MODE/hook: "hook"
+  # TODO: enable when non hook mode works
+  # INSTALL_MODE/secureboot: "secureboot"
+  # INSTALL_MODE/none: "none"
+  INSTALL_MODE/both: "both"
+
+  NESTED_UBUNTU_IMAGE_PRESEED_KEY/hook: "\" (test)\""
+  NESTED_ENABLE_TPM/hook: false
+  NESTED_ENABLE_SECURE_BOOT/hook: false
```

Comment:
Don't we need to set these 2 variables for "both" case? Is both SB+hook?

Reply:
"both" is sb + hook, yes. So NESTED_ENABLE_TPM and NESTED_ENABLE_SECURE_BOOT should have default value. Should I make the value explicit for all cases?

Reply:
It looks like the default for NESTED_ENABLE_SECURE_BOOT is not set by looking at spread.yaml, so I think we need to set it to true for "both" as a minimum

Reply:
Added definition. Also added a comment to make it more clear.

Reply:
Thanks!